doc/user-guides/multi-cluster.md

```diff
@@ -0,0 +1,289 @@
+EOF
+```
+
+RateLimitPolicy
+
+```
+kubectl apply -f  - <<EOF
+apiVersion: kuadrant.io/v1beta2
+kind: RateLimitPolicy
+metadata:
+  name: low-limit
+  namespace: ingress-gateway
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: external
+  overrides:
+    limits:
+      "low-limit":
+        rates:
+        - limit: 2
+          duration: 10
+          unit: second
+EOF
+```
+
+TLSPolicy
+
+```
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1alpha1
+kind: TLSPolicy
+metadata:
+  name: external
+  namespace: ingress-gateway
+spec:
+  targetRef:
+    name: external
+    group: gateway.networking.k8s.io
+    kind: Gateway
+  issuerRef:
+    group: cert-manager.io
+    kind: ClusterIssuer
+    name: lets-encrypt
+EOF
+```
+
+
+Lets check our policies have been accepted. 
+
+```
+kubectl get tlspolicy external -n ingress-gateway -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+
+kubectl get authpolicy external -n ingress-gateway -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+
+kubectl get ratelimitpolicy low-limit -n ingress-gateway -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+
+```
+Again we should see it has been accepted but not yet enforced. It is not enforced as we have not added any HTTPRoutes to the gateway yet.
+
+### Setup our DNS
+
+Next we will apply a `DNSPolicy`. This policy will configure how traffic reaches the gateways deployed to our different clusters. Again it will be accepted but not yet enforced as we have no HTTPRoutes defined at this point.
+
+```
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1alpha1
+kind: DNSPolicy
+metadata:
+  name: loadbalanced
+  namespace: ingress-gateway
+spec:
+  routingStrategy: loadbalanced
+  targetRef:
+    name: external
+    group: gateway.networking.k8s.io
+    kind: Gateway
+EOF
+```    
+Note: the DNSPolicy will leverage the ManagedZone we defined earlier based on the listener hosts defined in the gateway.
+
+check our status conditions
+```
+k get dnspolicy loadbalanced -n ingress-gateway -o=jsonpath='{.status.conditions[?(@.type=="Accepted")].message}'
+```
+
+You should see it has been accepted but not yet enforced.
+
+
+#TODO add section about viewing Gateway dashboards
+
+## RECAP
+
+So far we have setup an external gateway, secured it with TLS, Protected all endpoints with a default `DENY ALL` AuthPolicy added a restrictive RateLimitPolicy and set up ManagedZone and a DNSPolicy to ensure traffic is brought to the gateway for the listener hosts defined once we apply a HTTPRoute. Now that our policies are in place lets add a backend and HTTPRoute
+
+### Setup HTTPRoute and backend
+
+This will setup a toy application in the toystore ns
+
+```sh
+kubectl create ns toystore
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/main/examples/toystore/toystore.yaml -n toystore
+```
+
+### Setup HTTPRoute level RateLimits and Auth
```

Comment:
Section name mentions auth, but only RL is configured at the level of the route.